extensions/EXT_texture_filter_anisotropic/extension.xml

```diff
@@ -48,5 +52,8 @@ interface EXT_texture_filter_anisotropic {
     <revision date="2014/07/15">
       <change>Added NoInterfaceObject extended attribute.</change>
     </revision>
+    <revision date="2025/03/20">
+      <change>Note that WebGL 2 supports this extension's pnames in sampler entrypoints.</change>
```

Comment:
nit: Should be "pname" singular because only one of the two extension enums is accepted in sampler entrypoints.